README.md

```diff
@@ -157,13 +154,29 @@ Use the AWS CLI to sync the lambda code and demo console files to amazon S3:
 
  ```
    cd deployment/
-   aws s3 sync .dist/ s3://bucket-us-east-1/video-on-demand-on-aws/1.01/.
+   aws s3 sync dist/ s3://bucket-us-east-1/video-on-demand-on-aws/1.01
  ```
 
 ### 4. Launch the CloudFormation template.
 
 Launch the updated CloudFormation template from ```deployment/dist/``` folder.
 
+To deploy the stack for the first time with AWS CLI, modify the command below with you preferred settings.
+
+  ```
+  cd dist/
+  aws cloudformation deploy --template-file video-on-demand-on-aws.template --stack-name video-on-demand-on-aws --parameter-overrides AdminEmail=example@example.com WorkflowTrigger=MetadataFile Glacier=true FrameCapture=true --capabilities CAPABILITY_IAM
+
+  ```
```

Comment:
This took me a bit of hunting to actually get this deployed. Doing it via cloudformation console was constantly failing, but using AWS CLI worked fine.